### PR TITLE
commit

### PR DESCRIPTION
**Pull request by eaycayildirim**

### CODE REVIEW

File: Time Tracker/Button.cs [hunk 1]
```diff
@@ -10,19 +10,20 @@ public class Button
...
         public void Press()
         {
             //First two lines will change
 
             TimeSpan ts = _stopwatch.Elapsed;
             string elapsedTime = String.Format("{0:00}:{1:00}:{2:00}", ts.Hours, ts.Minutes, ts.Seconds);
-            _isPressed = !_isPressed;
-            if (this._isPressed)
+
+            IsPressed = !IsPressed;
```
Comment (garcorluis) [inline Time Tracker/Button.cs:26]: I don't like this line

File: Time Tracker/Program.cs [hunk 2]
```diff
@@ -11,12 +12,16 @@ static void Main(string[] args)
         {
             List<Button> buttons = new List<Button> { new Button() { Name = "Button 1" }, new Button() { Name = "Button 2" }, new Button() { Name = "Button 3" } };
             Tracker tracker = new Tracker(buttons);
+            CSV csv = new CSV();
```
Comment (garcorluis) [inline Time Tracker/Program.cs]: ????

File: Time Tracker/Program.cs [hunk 3]
```diff
@@ -11,12 +12,16 @@ static void Main(string[] args)
...
 
             tracker.Update();
+            csv.WriteLogsIntoCSV();
```
Comment (garcorluis) [inline Time Tracker/Program.cs]: ???

File: Time Tracker/Program.cs [hunk 4]
```diff
@@ -11,12 +12,16 @@ static void Main(string[] args)
...
             tracker.Update();
+            csv.WriteLogsIntoCSV();
```
Comment (garcorluis) [inline Time Tracker/Program.cs]: Why don't you include it into tracker?insideofl update?

File: Time Tracker/CSV.cs [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+using System.Text;
+using nsButton;
+using System.IO;
+
+namespace nsCSV
+{
+    class CSV : Button
+    {
+        private List<string> list = new List<string>();
+        public void WriteLogsIntoCSV()
```
Comment (garcorluis) [inline Time Tracker/CSV.cs]: think it again I think you can do it better . Print all the text at once maybe Its not best idea

Comment (garcorluis) [reply Time Tracker/CSV.cs]: Btw, the csu class shouldn't know what to write, only how and maybe where you want it to be written

File: TimeTrackerUI/App.config [hunk 6]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (garcorluis) [inline TimeTrackerUI/App.config:1]: I'm sure you didn't want to change this file

File: Time Tracker/Button.cs [hunk 7]
```diff
@@ -10,19 +10,20 @@ public class Button
...
+            if (this.IsPressed)
             {
                 _stopwatch.Start();
                 Console.WriteLine(Name + "; " + DateTime.Now + "; " + elapsedTime + " Started");
```
Comment (garcorluis) [inline Time Tracker/Button.cs]: will you priat it in the Console when you already have the CSV class?

File: Time Tracker/Button.cs [hunk 8]
```diff
@@ -10,19 +10,20 @@ public class Button
...
 
             TimeSpan ts = _stopwatch.Elapsed;
             string elapsedTime = String.Format("{0:00}:{1:00}:{2:00}", ts.Hours, ts.Minutes, ts.Seconds);
-            _isPressed = !_isPressed;
-            if (this._isPressed)
+
+            IsPressed = !IsPressed;
+            if (this.IsPressed)
             {
                 _stopwatch.Start();
```
Comment (garcorluis) [inline Time Tracker/Button.cs:29]: Here you start the stopwatch,but where do you stop it?

**Review by garcorluis — COMMENTED** [review]
Review the comments 😀

File: Time Tracker/Button.cs (hunk 1 above)
Comment (eaycayildirim) [reply Time Tracker/Button.cs:26]: It's almost same as before?

File: TimeTrackerUI/App.config (hunk 6 above)
Comment (eaycayildirim) [reply TimeTrackerUI/App.config:1]: Added because of File.Create command not by me

File: Time Tracker/Button.cs (hunk 1 above)
Comment (garcorluis) [reply Time Tracker/Button.cs:26]: Its the same. But I dont like any of them :D

**Pull request merged**